### ISSUE TITLE
Domains won't load, IP's will

### ISSUE THREAD

**Issue by JjTheHusky** [Genymobile/gnirehtet#189]
Hello, I received a LG G5 running Android 7, but the wifi, bluetooth and cellular has stopped working.
I found reverse tethering and hope it would work. I followed the instructions, but now when I try to goto a domain in chrome, it will display a `DNS_PROBE_FINISHED_NXDOMAIN` error. On the console, it will show `UdpConnection: 10.0.0.2:37488 -> 8.8.8.8:53 Open`

I figured it was a DNS error, I went to a direct IP in the browser and the page loaded. I looked around and saw I could do gnierhtet run -d x.x.x.x (DNS ip, I tried to use 1.1.1.1)
However this does not fix the error, and it appears to continue to use 8.8.8.8.

I think this might be the browser doing the DNS resolving instead of the network, however, I do not have the ability to do anything else with the device until the DNS works.
I'm not sure what to do from here.
Using Windows as OS.

**Comment by rom1v** [comment]
> I looked around and saw I could do gnierhtet run -d x.x.x.x (DNS ip, I tried to use 1.1.1.1)
> However this does not fix the error, and it appears to continue to use 8.8.8.8.

Did you stop the client on the device before running with a new DNS?

**Comment by JjTheHusky** [comment]
> > I looked around and saw I could do gnierhtet run -d x.x.x.x (DNS ip, I tried to use 1.1.1.1)
> > However this does not fix the error, and it appears to continue to use 8.8.8.8.
> 
> Did you stop the client on the device before running with a new DNS?

Yes, I did `gnierhtet stop` then `gnierhtet run -d 1.1.1.1` which gave the same result.

**Comment by rom1v** [comment]
I can't reproduce.

Which version of gnirehtet?
Which browser? (I tested with Firefox)
Does it happen with all apps?

**Comment by JjTheHusky** [comment]
This is the latest release v2.3 for windows 10 pro.
The phone has chrome version 52.0.2743.98
Unfortunately, it does not appear that any other app besides chrome will show network activity. The phone is basically vanilla android, so only Google apps are available.

**Comment by rom1v** [comment]
Is 1.1.1.1 accessible from your computer?

    dig @1.1.1.1 google.com

**Comment by JjTheHusky** [comment]
> ```
> dig @1.1.1.1 google.com
> ```

I'm not sure what dig is, but I think it's on Linux which I don't have. I pinged 1.1.1.1 and google.com and they are both accessible. I think the phone just wants to use 8.8.8.8 for some reason, which isn't wanting to connect.

**Comment by rom1v** [comment]
Are there any relevant logs in `adb logcat`, about the DNS associated to the VpnService?

**Comment by JjTheHusky** [comment]
Nothing is being logged by logcat, not sure if this is normal or not.
